src/config.ts

```diff
@@ -332,6 +332,13 @@ const notifications = {
     consumerSecret: envOrString(process.env.TWITTER_CONSUMER_SECRET),
     tweetTags: envOrString(process.env.TWITTER_TWEET_TAGS),
   },
+  streamlabs: {
+    accessToken: envOrString(process.env.STREAMLABS_ACCESS_TOKEN),
+    type: envOrString(process.env.STREAMLABS_TYPE),
+    imageHref: envOrString(process.env.STREAMLABS_IMAGE),
+    soundHref: envOrString(process.env.STREAMLABS_SOUND),
+    duration: envOrNumber(process.env.STREAMLABS_DURATION),
+  },
```

Comment:
```suggestion
  streamlabs: {
    accessToken: envOrString(process.env.STREAMLABS_ACCESS_TOKEN),
    duration: envOrNumber(process.env.STREAMLABS_DURATION),
    imageHref: envOrString(process.env.STREAMLABS_IMAGE),
    soundHref: envOrString(process.env.STREAMLABS_SOUND),
    text: {
        header: envOrString(process.env.STREAMLABS_HEADER),
        subheader: envOrString(process.env.STREAMLABS_SUBHEADER),
    },
    type: envOrString(process.env.STREAMLABS_TYPE),
  },
```

I'm wondering if we should take advantage of the `message` and `user_message` params?

Reply:
yeah, we could but as it for now the message parameter will be overwritten by the output of `Print.inStock (...)`.

I am thinking of two options:

1. View the product in the header and the store in the subheader
2. Display both the product and the store in the header and a custom message in the subheader (or vice versa)

Reply:
Oh! Right... Sorry, I totally forgot about that line. Okay, that sounds great. Let's just not change this for now and if we want to implement at some point, then we can. It's not a great improvement, so this is good to me!

Thanks for the help ❤️